K2Engineering/K2Engineering/DataTypes/BarData.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
```

Comment:
So here's where we're replacing (as I understand it) data-types with full-blown classes.

I've been asked for an opinion, so here it is. As I noted in above code review comments, I think it improves readability - at least as far as the code calls which are exemplified in this pull request.

This change *does* introduce a pretty hefty amount of LoC though. Always worth thinking to oneself "functionality is an asset, lines of code is a liability". In my opinion, if the developers' objective here is to subsequently make these classes (BarData, SupportData, RodData, PointLoadData) more intelligent, then that combined with improved readability is good justification for the code-base expansion. If it's only for the sake of readability, I'd be less enthusiastic with turning everything into classes.

Just my €0.02.